exasol/analytics/query/handler/udf/runner/state.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from exasol.analytics.schema.column import \
+    Column
+
+from exasol.analytics.query.handler.context.scope import     ScopeQueryHandlerContext
+from exasol.analytics.query.handler.context.top_level_query_handler_context import     TopLevelQueryHandlerContext
+from exasol.analytics.query.handler.query_handler     import QueryHandler
+from exasol.analytics.query.handler.udf.connection_lookup import UDFConnectionLookup
+
+
+@dataclass()
+class QueryHandlerRunnerState:
```

Comment:
```suggestion
class State:
```

Reply:
The current PR only renames files and directories. 
I propose to rename classes in the scope of the separate issue #211.